client/components/account-balances/index.tsx

```diff
@@ -26,6 +37,10 @@ const AccountBalances: React.FC = () => {
 	const { overviews, isLoading } = useAllDepositsOverviews();
 	const { selectedCurrency } = useSelectedCurrency();
 
+	const [ showInstantDepositNotice, setShowInstantDepositNotice ] = useState(
+		true
+	);
```

Comment:
> How to persist notice dismissal better
Tip: add a self-review comment about your concerns/considerations, e.g. here near the relevant code.

This dismissal needs to be persisted, merchants will be justifiably annoyed if they seet the notice after dismissing it. Feels like spam/ads.

Notices dismissals are usually persisted to store (woo) database, as user option or site option. This is a blocker, needs to be fixed.

Reply:
I have added the persistance via c9a35a2

Ref: discussion on slack where i got good guidance - p1718692277231899-slack-C02BW3Z8SHK